libs.js

```diff
@@ -0,0 +1,90 @@
+const PLANETS_LIB = {
+    orbital_period: 'Orbital Period',
+    climate: 'Climate',
+    diameter: 'Diameter',
+    gravity: 'Gravity',
+    surface_water: 'Surface Water',
+    terrain: 'Terrain',
+    population: 'Population',
+    rotation_period: 'Rotation Period'
+}
+const planetKeys = [
+    'orbital_period',
+    'climate',
+    'diameter',
+    'gravity',
+    'surface_water',
+    'terrain',
+    'population',
+    'rotation_period'
+];
+const PEOPLE_LIB = {
+    height: 'Height',
+    mass: 'Mass',
+    hair_color: 'Hair color',
+    skin_color: 'Skin color',
+    eye_color: 'Eye color',
+    birth_year: "Year of birth",
+    gender: 'Gender',
+}
+const peopleKeys = [
+    'height',
+    'mass',
+    'hair_color',
+    'skin_color',
+    'eye_color',
+    'birth_year',
+    'gender',
+];
```

Comment:
const peopleKeys = Object.keys(PEOPLE_LIB)

Reply:
Deleted all objects named smthKeys and replaced to Object.keys
Fixed